cms/core/viewsets.py

```diff
@@ -47,3 +49,53 @@ class ContactDetailsViewSet(SnippetViewSet):
 
     index_view_class = ContactDetailsIndex
     chooser_viewset_class = ContactDetailsChooserViewset
+
+
+class GlossaryTermCreateView(SnippetCreateView):
+    def save_instance(self) -> GlossaryTerm:
+        """Automatically set the last updating user on Glossary Term on creation and modification."""
+        instance: GlossaryTerm = super().save_instance()
+        instance.updated_by = self.request.user
+        instance.save(update_fields=["updated_by"])
+        return instance
+
+
+class GlossaryTermEditView(SnippetEditView):
+    def save_instance(self) -> GlossaryTerm:
+        """Automatically set the last updating user on Glossary Term on creation and modification."""
+        instance: GlossaryTerm = super().save_instance()
+        instance.updated_by = self.request.user
+        instance.save(update_fields=["updated_by"])
+        return instance
```

Comment:
non-blocking:

Looking at these, I wonder if https://docs.wagtail.org/en/stable/reference/hooks.html#after-edit-snippet and https://docs.wagtail.org/en/stable/reference/hooks.html#after-create-snippet are better in that we won't incur the extra db operation

Reply:
I'm happy to include this as part of this ticket - I'm just trying to make the functional tests work first...